### ISSUE TITLE
cannot connect to mpd running with bind_to_address "any"

### ISSUE THREAD

**Issue by jafz** [andrewrk/groovebasin#105]
Hi

some other issue i'm having is the following:
when running mpd with bind_to_address "any", groovebasin can't find the server.

So, i set bind_to to "localhost", restart groovebasin, and now it can find the server.
Bad thing is: i'd like to use other clients as well, like MpDroid, which requires the mpd server to be run with bind_to "any", so outside clients can connect.

So, set mpd to "any" again, restart mpd.
MpDroid works again.
groovebasin, which was started while mpd was running under bind_to "localhost" can still connect to mpd ("any"). However, restarting groovebasin now leads again to the error that it can't find the mpd daemon.

In a nutshell: there seems to be an issue when starting groovebasin. Once it's running, it's compliant.

Now, why is that?

Cheers
Jakob
